dluhc-component-library/src/components/maps/baseMap.tsx

```diff
@@ -59,6 +61,11 @@ const BaseMap = ({
     );
     map.setTarget(ref.current);
     map.addInteraction(new Draw({ source: source, type: "Polygon" }));
+    source.on("addfeature", async function (evt) {
```

Comment:
Lambda function added

As far as I can tell, it works as I intend. No new event listeners are created when I check via getEventListeners(window) in the dev console. Happy to chat more about this

Reply:
Interesting, i wonder if its because we recreate the source each render. Still feels fishy to me but not something to block the PR if it looks okay from tests.